README.md

```diff
@@ -5,6 +5,7 @@
 - TODO: Write a detailed description of the project here.
 - TODO: Add description of project to projects section of website.
 - TODO: Create a slack channel for the project.
+- TODO: Create a page in the [minutes Google Doc][minutes] for the project.
```

Comment:
@redisforlosers 

This process is pretty straightforward but I will outline the couple of steps involved just in case Future CFFC forgets. I would also like to indicate that someone with Edit permissions has to do it. I am open to providing the permissions to each project lead, but since it's a one-time deal for each project and we don't have a ton of new projects being added often, it might be easier if one of us just does it (it only took you and me a couple minutes to knock it out on Thursday).